### ISSUE TITLE
Error when trying to use /rclc add <item>

### ISSUE THREAD

**Issue by steino** [evil-morfar/RCLootCouncil_Classic#33]
Hi. When trying to use /rclc add <item> I get the error below. It seems to happen after login/reload and it might be cause OnEnable is not being called. I'm not quite familiar with the code to see exactly what's causing this, but one this I noticed is that I'm never asked if I want to use RCLC when I become the ML.
```lua
1x ...dOns\RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:100: attempt to get length of field 'lootTable' (a nil value)
...dOns\RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:100: in function `AddItem'
...dOns\RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:345: in function `AddUserItem'
RCLootCouncil_Classic\RCLootCouncil\core.lua:991: in function `ChatCmdAdd'
RCLootCouncil_Classic\RCLootCouncil\core.lua:422: in function `?'
Ace3\AceConsole-3.0\AceConsole-3.0-7.lua:94: in function `?'
FrameXML\ChatFrame.lua:4695: in function `ChatEdit_ParseText'
FrameXML\ChatFrame.lua:4356: in function <FrameXML\ChatFrame.lua:4355>

Locals:
self = <table> {
 SetDefaultModuleLibraries = <function> defined @Ace3\AceAddon-3.0\AceAddon-3.0.lua:367
 LootOnClick = <function> defined @RCLootCouncil_Classic\API\MLUpdates.lua:183
 EnableModule = <function> defined @Ace3\AceAddon-3.0\AceAddon-3.0.lua:332
 modules = <table> {
 }
 CancelTimer = <function> defined @Ace3\AceTimer-3.0\AceTimer-3.0.lua:147
 IterateEmbeds = <function> defined @Ace3\AceAddon-3.0\AceAddon-3.0.lua:442
 baseName = "RCLootCouncil_Classic"
 SecureHook = <function> defined @Ace3\AceHook-3.0\AceHook-3.0.lua:336
 PrintAwardedInBags = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:405
 HookLootButton = <function> defined @RCLootCouncil_Classic\API\MLUpdates.lua:165
 LootOpened = <function> defined @RCLootCouncil_Classic\API\MLUpdates.lua:118
 LootTableCompare = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1715
 RawHookScript = <function> defined @Ace3\AceHook-3.0\AceHook-3.0.lua:395
 SortLootTable = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1689
 EQUIPLOC_SORT_ORDER = <table> {
 }
 AwardPopupOnClickNo = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1648
 AwardPopupOnClickYes = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1639
 SetDefaultModulePrototype = <function> defined @Ace3\AceAddon-3.0\AceAddon-3.0.lua:409
 GetItemsFromMessage = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1510
 AwardPopupOnShow = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1618
 OnCommReceived = <function> defined @RCLootCouncil_Classic\Core\Comms.lua:90
 defaultModuleState = true
 SendWhisperHelp = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1595
 IsEnabled = <function> defined @Ace3\AceAddon-3.0\AceAddon-3.0.lua:451
 RemoveItem = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:147
 ScheduleTimer = <function> defined @Ace3\AceTimer-3.0\AceTimer-3.0.lua:94
 DisableModule = <function> defined @Ace3\AceAddon-3.0\AceAddon-3.0.lua:350
 announceItemStringsDesc = <table> {
 }
 AwardPopupOnClickYesCallback = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1624
 GiveLoot = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:897
 AnnounceAward = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1265
 RegisterMessage = <function> defined @Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:90
 UnregisterMessage = <function> defined @Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:153
 RegisterBucketEvent = <function> defined @Ace3\AceBucket-3.0\AceBucket-3.0.lua:184
 AnnounceItems = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1203
 IsHooked = <function> defined @Ace3\AceHook-3.0\AceHook-3.0.lua:493
 OnBonusRoll = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:662
 IsItemIgnored = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1482
 SendCommMessage = <function> defined @Ace3\AceComm-3.0\AceComm-3.0.lua:87
 Unhook = <function> defined @Ace3\AceHook-3.0\AceHook-3.0.lua:418
 GetCouncilInGroup = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:1491
 OnLootSlotCleared = <function> defined @RCLootCouncil_Classic\RCLootCouncil\ml_core.lua:739
 UnregisterAllComm = <function> defined @Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:174
 SetEnabledState = <function> defined @
```

**Comment by evil-morfar** [comment]
Not being asked if you'd want to use the addon could very well cause this error.
I'd need to have a look at your saved variables.

**Comment by steino** [comment]
After some further testing it seems this only happens if you're in a party as the master looter and trying to use /rclc add. We encountered this error when doing some testing outside of raids. The popup asking of I want to use rclc does work fine when its a raid, but not when I'm in a party.

**Comment by evil-morfar** [comment]
Have you enabled `Only use raids`?

**Comment by steino** [comment]
I tested with fresh savedvariables and "Only use in raids" seems to be enabled by default.

**Comment by evil-morfar** [comment]
So let me rephrase: have you disabled it?

**Comment by steino** [comment]
I tested with setting disabled and yes the popup happens and clicking yes to enable rclc for the raid makes the error go away. The error occours if you try to use /rclc add before any rclc session has been started, either starting it normally or via /rclc test after a reload or login.

**Comment by evil-morfar** [comment]
So with it disabled, if you do a reload in a group, get the pop-up, click yes, and try to start a session you get the error?
